openwisp_notifications/apps.py

```diff
@@ -7,9 +8,19 @@ class OpenwispNotificationsConfig(AppConfig):
     verbose_name = _('Notifications')
 
     def ready(self):
-        from openwisp_notifications.signals import notify  # noqa
+        from openwisp_notifications.signals import notify
         from openwisp_notifications.handlers import notify_handler
 
         notify.connect(
             notify_handler, dispatch_uid='openwisp_notifications.model.notifications'
         )
+        self.add_default_menu_items()
+
+    def add_default_menu_items(self):
+        menu_setting = 'OPENWISP_DEFAULT_ADMIN_MENU_ITEMS'
+        items = [{'model': 'openwisp_notifications.Notification'}]
+        if not hasattr(settings, menu_setting):
+            setattr(settings, menu_setting, items)
+        else:
+            current_menu = getattr(settings, menu_setting)
+            current_menu += items
```

Comment:
I did not spot this, please remove it, we don't need it. We have the bell icon and we'll have the notification JS widget for this.